tests/conftest.py

```diff
@@ -72,11 +74,44 @@ def compile_and_run(device, reset_torch_dynamo, request):
             accuracy = calculate_accuracy(outputs, outputs_after)
             if accuracy:
                 comp_runtime_metrics["accuracy"] = accuracy
+
+            # Memory analysis
+            mm = option._memory_manager
```

Comment:
probably not a private member anymore if you access it from outside like this
```suggestion
            mm = option.memory_manager
```

Reply:
Yeah will fix it.